backend/app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Admin
+  class UsersController < ApplicationController
+    before_action :check_admin
```

Comment:
Nits

createアクションしかないですが、onlyオプション渡しましょか
```suggestion
    before_action :check_admin, only: %i[create]
```